README.md

```diff
@@ -165,6 +176,11 @@ To run all the tests you can execute the following:
    npm start
    ```
    Also keep this process running.
+4. Start the http proxy
```

Comment:
Step 4 comes before step 3.

Reply:
Corrected this order in [commit 03f3f2d](https://github.com/SolidLabResearch/generic-data-viewer/pull/78/commits/03f3f2d5595768b8fab7dc96bcf9d308aa8e1428)